pkg/forwarder/forwarder.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/version"
 )
 
+const (
+	// PayloadTypePod is the name of the pod payload type
+	PayloadTypePod = "pod"
```

Comment:
maybe you can introduce `type PayloadType string`

```suggestion
	PayloadTypePod PayloadType = "pod"
```